frontend/packages/ceph-storage-plugin/src/components/namespace-store/namespace-store-table.scss

```diff
@@ -0,0 +1,46 @@
+.nb-ns-table {
+  .pf-c-toolbar__content {
+    padding-left: 0;
+  }
+
+  .pf-c-table {
+    border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
+  }
+
+  .co-m-pane__body {
+    padding: 0;
+  }
+}
+
+.nb-bc-step-page-form {
+  .pf-c-alert.pf-m-inline.pf-m-info.co-alert {
+    margin-bottom: 0;
+  }
+  &--margin {
+    margin-bottom: var(--pf-global--spacer--lg);
+  }
+}
+
+.nb-ns__namespacestore-table {
```

Comment:
can you make a comment that it is deduplicated from `.ceph-ocs-install__select-nodes-table `
So if any change happen we must fix here plus later we must import it from a common scss file.

Reply:
added comment